crates/bevy_image/src/texture_atlas.rs

```diff
@@ -51,14 +51,27 @@ impl TextureAtlasSources {
         })
     }
 
-    /// Retrieves the texture *section* rectangle of the given `texture` handle.
+    /// Retrieves the texture *section* rectangle of the given `texture` handle in pixels.
     pub fn texture_rect(
         &self,
         layout: &TextureAtlasLayout,
         texture: impl Into<AssetId<Image>>,
     ) -> Option<URect> {
         layout.textures.get(self.texture_index(texture)?).cloned()
     }
+
+    /// Retrieves the texture *section* rectangle of the given `texture` handle in UV coordinates.
```

Comment:
For folks who are new to game dev, another line about what UV coordinates means would be helpful.

Reply:
Not a native English speaker. Does this read okay?
> Retrieves the texture *section* rectangle of the given `texture` handle in UV coordinates.
> These are within the range [0..1], as a fraction of the entire texture atlas's size.

Reply:
Yeah, that's much better!